src/pkg_tsl2b/word_processors.py

```diff
@@ -98,59 +97,84 @@ def count_words(text: str) -> Dict[str, int]:
         none
 
     Restrictions on when this method can be called:
-        text should be cleanish.
+        text for counting words should be cleanish.
     '''
 
-    assert isinstance(text, str)
+    assert isinstance(text_for_counting_words, str)
 
-    list_of_words = tokenize(text)
+    list_of_words = tokenize(text_for_counting_words)
     the_counter = Counter(list_of_words)
-    dictionary_of_words_and_counts = dict(the_counter)
+    dictionary = dict(the_counter)
 
-    #logger.info("Counted words: %s", dictionary_of_words_and_counts)
+    #logger.info("Counted words: %s", dictionary)
 
-    assert not dictionary_of_words_and_counts is None
-    assert isinstance(dictionary_of_words_and_counts, dict)
+    assert not dictionary is None
+    assert isinstance(dictionary, dict)
 
-    return dictionary_of_words_and_counts
+    return dictionary
 
 
 def parse_arguments():
-    dictionary_of_arguments = {}
-    parser = argparse.ArgumentParser(prog = "Word Processors", description = "This program processes text.")
+    '''
+    Parses command line arguments into a dictionary of argument names and values
+
+    Keyword arguments:
+        none
+
+    Return values:
+        dictionary_of_arguments: Dict[str, int] -- dictionary of argument names and values
+
+    Side effects:
+        none
+
+    Exceptions raised:
+        none
+
+    Restrictions on when this method can be called:
+        first when this script is executed
+    '''
+
+    dictionary = {}
+    parser = argparse.ArgumentParser(
+        prog = "Word Processors",
+        description = "This program processes text."
+    )
     parser.add_argument("name_of_function", help = "name of function")
     parser.add_argument("path_to_file_of_text", help = "path to file of text")
     args = parser.parse_args()
-    name_of_function = args.name_of_function
-    path_to_file_of_text = args.path_to_file_of_text
-    print(f"name of function: {name_of_function}")
-    print(f"path to file of text: {path_to_file_of_text}")
-    dictionary_of_arguments["name_of_function"] = name_of_function
-    dictionary_of_arguments["path_to_file_of_text"] = path_to_file_of_text
-    return dictionary_of_arguments
+    function_name = args.name_of_function
```

Comment:
This passage could use a little 'phrasing'...  Again, another thing that is not gospel, but when I read it I had to strain a bit to grok it.  

For example

```python
    parser.add_argument("name_of_function", help = "name of function")
    parser.add_argument("path_to_file_of_text", help = "path to file of text")
    args = parser.parse_args()

    function_name = args.name_of_function
    path_to_text_file = args.path_to_file_of_text
    print(f"name of function: {function_name}")
    print(f"path to file of text: {path_to_text_file}")

    dictionary["name_of_function"] = function_name
    dictionary["path_to_file_of_text"] = path_to_text_file

    return dictionary
```

would clearly separate the thoughts and jumps out at you, making it easier to read in my opinion
* parse
* extract
* build dictionary
* return the dictionary

Reply:
@EfrainOlivaresUVA , I phrased.